.github/gh_matrix_builder.py

```diff
@@ -130,9 +127,8 @@ def build_apache_config(overrides):
             "coverage": False,
         }
     )
-    base_config.update(apache_config)
-    base_config.update(overrides)
-    return base_config
+    apache_config.update(overrides)
+    return build_debug_config(apache_config)
 
```

Comment:
This seems like an unrelated change and should probably be moved into separate PR

Reply:
https://github.com/timescale/timescaledb/pull/7562